Server/Services/auth.py

```diff
@@ -0,0 +1,26 @@
+from DB_Connection.adminQueries import AdminQuery
+
+class Authentication:

```

Comment:
I would name the file name and class name as AuthenticationService

Reply:
I have renamed the name of file as well as class to AuthenticationService